src/typography/typography.stories.html

```diff
@@ -21,7 +21,20 @@ <h3>Code example: Font</h3>
     </div>
   </section>
   <section>
-    <h3>Code examples: text decoration</h3>
+    <h3>Code example: Serif</h3>
+    <p>OpenMRS uses merriweather for serif style.</p>
```

Comment:
Maybe modify line 9 of this file to clarify that Roboto applies by default to all text, but Merriweather is also available?

Reply:
Done